worlds/civ_6/__init__.py

```diff
@@ -0,0 +1,192 @@
+import math
+import os
+from typing import Any, Dict, Optional, Set
+
+from .Data import get_boosts_data
+
+from .Rules import create_boost_rules
+from .Container import CivVIContainer, generate_goody_hut_sql, generate_new_items, generate_setup_file, generate_update_boosts_sql
+from .Enum import CivVICheckType
+from .Items import BOOSTSANITY_PROGRESSION_ITEMS, FILLER_DISTRIBUTION, CivVIItemData, FillerItemRarity, generate_item_table, CivVIItem, get_random_filler_by_rarity
+from .Locations import CivVILocation, CivVILocationData, EraType, generate_era_location_table, generate_flat_location_table
+from .Options import CivVIOptions
+from .Regions import create_regions
+from BaseClasses import Item, ItemClassification, MultiWorld, Tutorial
+from worlds.AutoWorld import World, WebWorld
+from worlds.LauncherComponents import Component, SuffixIdentifier, Type, components, launch_subprocess
+
+
+def run_client(url: Optional[str] = None):
+    print("Running Civ6 Client")
+    from .Civ6Client import main  # lazy import
+    launch_subprocess(main, name="Civ6Client")

```

Comment:
```suggestion
def run_client():
    print("Running Civ6 Client")
    from .Civ6Client import main  # lazy import
    launch_subprocess(main, name="Civ6Client")
```
Unused

Reply:
So if I try to run the client via the `.apcivvi` file then I end up getting this error when I remove the URL:
`TypeError: run_client() takes 0 positional arguments but 1 was given`. Is this expected, or am I doing something else incorrectly?

Reply:
It looks like there may be a different way for non-apworld clients:
https://github.com/hesto2/Archipelago/blob/civ6-1.0/worlds/LauncherComponents.py#L161
I'll follow this pattern and drop it there and probably move the Civ6 client to root like the others. Definitely check me if that seems off though 🙏

Reply:
Ok just updated, take a look here: https://github.com/ArchipelagoMW/Archipelago/pull/3736/files#diff-149bb8d36d711ea121669f2c2787d83e31ee7e7fd680e3f8835597c8cbc09927

I verified this works as expected still as far as opening the client through the launcher and doing the `open patch` with the `.apcivvi` 👍